Makefile

```diff
@@ -10,7 +10,7 @@ clean:
 	@rm -rf controller/controller
 
 build:
-	@cd controller && godep go build -a -tags "netgo static_build" -installsuffix netgo -ldflags "-w -X github.com/shipyard/shipyard/version.GitCommit=$(COMMIT)" .
+	@cd controller && godep go build -a -tags "netgo static_build" -installsuffix netgo -ldflags "-w -X github.com/shipyard/shipyard/version.GitCommit $(COMMIT)" .
```

Comment:
So, can I say our Makefile is not compatible to Go version before 1.5? What about making it compatible?